lib/ledge.lua

```diff
@@ -176,6 +176,11 @@ _M.create_redis_slave_connection = create_redis_slave_connection
 
 
 local function close_redis_connection(redis)
+    if not next(redis) then
+        -- Possible for this to be called before we've created a redis conn
+        -- Ensure we actually have a resty-redis instance to close
+        return nil
+    end
```

Comment:
Minor style thing; usually leave a space after a control structure block. There are exceptions, but I try to keep it so that branches read like their own paragraphs.